debt-token/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+#![deny(warnings)]
+#![no_std]
+
+mod event;
+mod test;
+
+use common_token::{
+    balance::{add_total_supply, receive_balance, spend_balance},
+    check_nonnegative_amount,
+    storage::*,
+    verify_caller_is_pool,
+};
+use debt_token_interface::DebtTokenTrait;
+use soroban_sdk::{contractimpl, Address, Bytes, Env};
+use soroban_token_sdk::TokenMetadata;
+
+pub struct DebtToken;
+
+#[contractimpl]
+impl DebtTokenTrait for DebtToken {
+    fn initialize(
+        e: Env,
+        decimal: u32,
+        name: Bytes,
+        symbol: Bytes,
+        pool: Address,
+        underlying_asset: Address,
+    ) {
+        if decimal > u8::MAX.into() {
+            panic!("Decimal must fit in a u8");
```

Comment:
I see that sometimes we start the panic statements with capital letter. Could you make it consistent with other "panics"?